tests/cypress/fixtures/graphql/jcrDeleteProperty.graphql

```diff
@@ -0,0 +1,9 @@
+mutation($pathOrId: String!, $property: String!, $language: String){
```

Comment:
What do you think of moving this file the @jahia/cypress module, it could be used for other tests